android/src/BluetoothUuids.java

```diff
@@ -24,6 +24,18 @@ public interface BluetoothUuids {
   UUID HEART_RATE_MEASUREMENT_CHARACTERISTIC =
     UUID.fromString("00002A37-0000-1000-8000-00805F9B34FB");
 
+  /**
+   * Engine sensors service
+   */
+  UUID ENGINE_SENSORS_SERVICE =
+    UUID.fromString("D2865ECA-2C07-4610-BF03-8AEEBEF047FB");
+
+  /**
+   * Engine sensors characteristic
+   */
```

Comment:
This needs a more extensive code comment. This is not a generic "Engine sensor", but your proprietary product. There should be a description of your product or a link to your website which describes the product.